src/cartridges/bm_adyen/cartridge/static/default/adyenSettings.js

```diff
@@ -86,6 +90,76 @@ document.addEventListener('DOMContentLoaded', () => {
     this.classList.toggle('bi-eye');
   }
 
+  // open Adyen Giving Background upload page
+  function uploadAdyenGivingBackground() {
+    const params = 'resizable=no,width=1000,height=500,left=100,top=100';
+    const openedWindow = window.open(
+      window.adyenGivingBackgroundUrl,
+      'test',
+      params,
+    );
+    const loop = setInterval(() => {
+      if (openedWindow.closed) {
+        window.location.reload();
+        clearInterval(loop);
+      }
+    }, 1000);
+  }
+
+  // open Adyen Giving Logo upload page
+  function uploadAdyenGivingLogo() {
+    const params = 'resizable=no,width=1000,height=500,left=100,top=100';
+    const openedWindowLogo = window.open(
+      window.adyenGivingLogoUrl,
+      'test',
+      params,
+    );
+    const loop = setInterval(() => {
+      if (openedWindowLogo.closed) {
+        window.location.reload();
+        clearInterval(loop);
+      }
+    }, 1000);
+  }
+
+  function getImageName(imageUrl) {
+    const parts = imageUrl.split('/');
+    const imageName = parts.pop();
+    return imageName;
+  }
+
+  function printBackgroundImageName() {
+    document.getElementById('backgroundFileName').innerHTML = '';
+    const ul = document.getElementById('backgroundFileName');
+    const nameOfImage = getImageName(window.backgroundValueField);
+    if (nameOfImage?.length > 0) {
```

Comment:
the contents of this if block are a bit difficult to parse.
Can you maybe change some variable names or add some comments to make it more readable?